### PR TITLE
Add working directory argument for local-cluster

### PR DESCRIPTION
**Pull request by MitchyCola**
Provides an option to store the data of a running local-cluster into a specified working directory.

### CODE REVIEW

File: local-cluster/Main.hs [hunk 1]
```diff
@@ -23,7 +26,9 @@ import Test.Plutip.LocalCluster (
 main :: IO ()
 main = do
   args <- Options.execParser (info (pClusterConfig <**> helper) mempty)
-  (st, _) <- startCluster def $ do
+  let workingDir = maybe Temporary (flip Fixed False) (dirWorking args)
+      plutipConfig = PlutipConfig Nothing Nothing Nothing 1 workingDir []
```
Comment (mikekeke) [inline local-cluster/Main.hs:30]:
```suggestion
      plutipConfig = def {clusterWorkingDir = workingDir}
```

File: local-cluster/Main.hs [hunk 2]
```diff
@@ -108,5 +121,6 @@ data CWalletConfig = CWalletConfig
   , adaAmount  :: Integer
   , lvlAmount  :: Integer
   , numUtxos :: Int
+  , dirWorking :: Maybe FilePath
```
Comment (mikekeke) [inline local-cluster/Main.hs:124]: Maybe name it also something like `workDir` or `clusterWorkingDir`

Comment (MitchyCola) [reply local-cluster/Main.hs]: Changed to `workDir`, since `clusterWorkingDir` is being used by `PlutipConfig`.

**Comment by mikekeke** [comment]
👍

**Review by mikekeke — APPROVED** [review]

**Pull request merged**